lib/active_model_serializers/adapter/attributes.rb

```diff
@@ -8,7 +8,7 @@ def initialize(serializer, options = {})
       end
 
       def serializable_hash(options = nil)
-        options ||= {}
+        options = super
```

Comment:
Super does return a hash. We need to make sure options is always set.  This is how it is in Rails.  See links in https://github.com/rails-api/active_model_serializers/issues/1643. If someone passes in 'nil', it sets options to '{}'.  Rather than make sure the signature is always 'options = nil' and the first line in the method 'options ||= {}', we just delegate that guard to the base class.  Alternatively, we use a different abstract method that serializable_hash calls.